add a result html documentation to the readme file (#1122)
test-network-function/cnf-certification-test: README.md

CNF Certification Test is copyright [Red Hat, Inc.](https://www.redhat.com) and available
under an
[Apache 2 license](https://github.com/test-network-function/cnf-certification-test/blob/main/LICENSE).

## Show Results after finishing of running the test code

After the end of your run a claim.json file will be created for you with the
results of that specific test run. To see them in a good way that is clear for
you we created a parser that is local. You can open it by copying the path /

>  #your-code-path/cnf-certification-test/script/results html and open it in your

Reviewer: Shouldn't it be ".../results.html" instead of "./results html".

CNF Certification Test is copyright [Red Hat, Inc.](https://www.redhat.com) and available
under an
[Apache 2 license](https://github.com/test-network-function/cnf-certification-test/blob/main/LICENSE).

> ## Show Results after finishing of running the test code

Reviewer 2: ```suggestion
## Show Results after finishing of running the test code

```

> after the end of your run a claim.json file will be created for you with the results of that spcific running. to see them in a good way that is clear for you we created a result url page that is local you can open it by copy that path /#your-code-path/cnf-certification-test/script/results html and open it on bowser then upload the calim.json file of that run

Reviewer 3: ```suggestion
After the end of your run a claim.json file will be created for you with the results of that specific test run. To see them in a good way that is clear for you we created a parser that is local. You can open it by copying the path /#your-code-path/cnf-certification-test/script/results html and open it in your browser then upload the claim.json file of that run.
```

> After the end of your run a claim.json file will be created for you with the results of that specific test run. To see them in a good way that is clear for you we created a parser that is local. You can open it by copying the path /#your-code-path/cnf-certification-test/script/results html and open it in your browser then upload the claim.json file of that run.

Reviewer: You should also remark that, in order for the "results.html" file to parse and render the claim.json file correctly, this other file need to be copied in the same folder as the "results.html" file.
- script/classification.js : static file that can be found in the repo.

Author: the file is there so i need to do that?

Reviewer: @aabughosh , please, disregard my comment. They're both actually inside the same "scripts" folder.

> After the end of your run a claim.json file will be created for you with the results of that specific test run. To see them in a good way that is clear for you we created a parser that is local. You can open it by copying the path /#your-code-path/cnf-certification-test/script/results html and open it in your browser then upload the claim.json file of that run.

Reviewer 2: ```suggestion
After the end of your run a claim.json file will be created for you with the                                                           
results of that specific test run. To see them in a good way that is clear for                                                         
you we created a parser that is local. You can open it by copying the path                                                             
/#your-code-path/cnf-certification-test/script/results html and open it in your                                                        
browser then upload the claim.json file of that run.
```
